app/src/main/java/com/quran/labs/androidquran/common/LocalTransationDiplaySort.kt

```diff
@@ -0,0 +1,8 @@
+package com.quran.labs.androidquran.common
+
+class LocalTransationDiplaySort : Comparator<LocalTranslation> {
+  override fun compare(p0: LocalTranslation?, p1: LocalTranslation?): Int {
```

Comment:
do we really expect these to be nullable? because if not, we can simplify things here

Reply:
yup, good call